run_me.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
-python main.py
-python test_parse_fasta.py
-python test_sam_to_fasta_file.py
+python src/main.py
```

Comment:
I have reorganized the modules so that all code is in `src/` and `tests/` (for test code).